cloud/blockstore/libs/storage/partition_nonrepl/multi_agent_write_actor.h

```diff
@@ -151,12 +182,39 @@ template <typename TMethod>
 void TMultiAgentWriteActor<TMethod>::SendWriteRequest(
     const NActors::TActorContext& ctx)
 {
+    size_t readyForMultiagentWrite = CountIf(
+        ReplicasDiscovery,
+        [](const TReplicaDiscovery& replica)
+        { return replica.ReadyForMultiAgentWrite; });
+
+    if (readyForMultiagentWrite < 2) {
+        // Only one partition can perform direct disk-agent write.
+        // Let's fallback to TMirrorRequestActor<> actor.
+        Fallback(ctx);
```

Comment:
By the way why fallback to mirror actor when there is only one multi agent capable partition. Why not use `SendOrdinaryWriteRequests` for the not capable partitions here as well ?